### PR TITLE
TexturePreview Update text when texture is being changed

### LINKED ISSUE #53023: When changing region of `AtlasTexture` the size in the texture preview isn't updated
### Godot version

3.4.beta5

### System information

N/A

### Issue description

Title. In action:
Fiav10RSP9

### Steps to reproduce

Edit `AtlasTexture`'s region using _TextureRegion_ bottom panel.

### Minimal reproduction project

_No response_

### PR DESCRIPTION
**Pull request by kleonc**
Fixes #53023.

### CODE REVIEW

**Review by akien-mga — APPROVED** [review]

File: editor/plugins/texture_editor_plugin.cpp [hunk 1]
```diff
@@ -58,6 +58,25 @@ void TexturePreview::_notification(int p_what) {
...
+	} else {
+		format = texture->get_class();
+	}
+
+	metadata_label->set_text(itos(texture->get_width()) + "x" + itos(texture->get_height()) + " " + format);
+}
+
+void TexturePreview::_bind_methods() {
+	ClassDB::bind_method(D_METHOD("_update_metadata_label_text"), &TexturePreview::_update_metadata_label_text);
+}
```
Comment (akien-mga) [inline editor/plugins/texture_editor_plugin.cpp:78]: You don't need to bind the callback in 4.0, `callable_mp` doesn't need it.

Comment (kleonc) [reply editor/plugins/texture_editor_plugin.cpp]: Oh, good to know! Removed it already.
Is binding needed in some situations though? I can still see binds to private methods here and there, I wonder if they're just leftovers or if sometimes they're actually needed.

**Comment by akien-mga** [comment]
Thanks!

File: editor/plugins/texture_editor_plugin.cpp (hunk 1 above)
Comment (akien-mga) [reply editor/plugins/texture_editor_plugin.cpp]: They're still needed for stuff like UndoRedo which can't rely on Callables currently.

**Pull request merged**